src/new-apps/components/InstalledAppListRow/InstalledAppListRow.tsx

```diff
@@ -1,81 +1,83 @@
+import Link from "@dashboard/components/Link";
 import { TableButtonWrapper } from "@dashboard/components/TableButtonWrapper/TableButtonWrapper";
-import TableCellAvatar from "@dashboard/components/TableCellAvatar";
-import TableRowLink from "@dashboard/components/TableRowLink";
 import { useAppListContext } from "@dashboard/new-apps/context";
 import { appsMessages } from "@dashboard/new-apps/messages";
 import { InstalledApp } from "@dashboard/new-apps/types";
 import { AppUrls } from "@dashboard/new-apps/urls";
 import { isAppInTunnel } from "@dashboard/new-apps/utils";
-import { TableCell, Typography } from "@material-ui/core";
-import { IconButton, Pill, SettingsIcon } from "@saleor/macaw-ui";
-import clsx from "clsx";
+import {
+  Box,
+  Button,
+  Chip,
+  List,
+  sprinkles,
+  Text,
+} from "@saleor/macaw-ui/next";
 import React from "react";
-import { useIntl } from "react-intl";
+import { FormattedMessage, useIntl } from "react-intl";
 
-import AppManifestTableDisplay from "../AppManifestTableDisplay";
-import { AppPermissions } from "../AppPermissions";
+import { AppAvatar } from "../AppAvatar/AppAvatar";
+import AppPermissions from "../AppPermissions";
 import { messages } from "./messages";
-import { useStyles } from "./styles";
 
 export const InstalledAppListRow: React.FC<InstalledApp> = props => {
   const { app, isExternal, logo } = props;
   const intl = useIntl();
-  const classes = useStyles(props);
   const { openAppSettings } = useAppListContext();
 
   return (
-    <TableRowLink className={classes.row} href={AppUrls.resolveAppUrl(app.id)}>
-      <TableCellAvatar
-        initials={app.name?.[0]?.toUpperCase()}
-        thumbnail={logo?.source || undefined}
-        avatarProps={classes.logo}
-        className={clsx(classes.col, classes.colLogo)}
+    <Link
+      href={AppUrls.resolveAppUrl(app.id)}
+      className={sprinkles({ display: "contents" })}
+      inline={false}
+    >
+      <List.Item
+        padding={7}
+        borderTopStyle="solid"
+        borderWidth={1}
+        borderColor="neutralPlain"
+        className={sprinkles({
```

Comment:
**suggestion**: I think you can use `justifyContent` and `flexDirection` directly on [List.Item](https://github.com/saleor/macaw-ui/blob/e9dd887c829e8a78b17ac43ebda217357925fa14/src/components/List/Item.tsx#L6)